.evergreen.yml

```diff
@@ -56,43 +140,73 @@ functions:
       binary: npm
       args: [run, coverage]
 
-  attach-results:
+  run-cypress-tests:
+    command: shell.exec
+    params:
+      working_dir: spruce
+      script: |
+        Xvfb :1 -screen 0 1280x800x24 &
+        export DISPLAY=:1
```

Comment:
this is a workaround for a known bug for Cypress running on Ubuntu 18.04. Version 3.8.3 of Cypress fixes the issue.